cypress/integration/navigation.spec.js

```diff
@@ -2,13 +2,16 @@ describe("Navigation bar", () => {
     beforeEach(() => {
         cy.setCookie("Other", "other");
         cy.setCookie("XSRF-TOKEN", "abcde");
-        cy.visit("/supervision/deputies/public-authority/deputy/1/");
+        cy.visit("/supervision/deputies/public-authority/deputy/1");
     });
 
     const expected = [
-        ["Dashboard", "/supervision/deputies/public-authority/deputy/1/"],
+        ["Dashboard", "/supervision/deputies/public-authority/deputy/1"],
         ["Clients", "/supervision/deputies/public-authority/deputy/1/clients"],
-        ["Timeline", "/supervision/deputies/public-authority/deputy/1/timeline"],
+        [
+            "Timeline",
+            "/supervision/deputies/public-authority/deputy/1/timeline",
```

Comment:
is it worth removing the trailing comma and keeping this to one line for consistency?

Reply:
This is auto-formatted according to the Prettier rules. It is possible to change the suggested line length but it isn't advised apparently.